Pod/ILGClasses/ILGClasses.m

```diff
@@ -66,9 +66,20 @@ + (NSSet *)subclassesOfClass:(Class)superclass
 
 + (NSSet *)classesConformingToProtocol:(Protocol *)protocol
 {
-    return [self classesPassingTest:^BOOL(Class class) {
+    NSSet *superclassesConforming = [self classesPassingTest:^BOOL(Class class) {
         return class_conformsToProtocol(class, protocol);
     }];
+    
+    NSMutableSet *subclasses = [NSMutableSet set];
+    
+    for (Class superclass in superclassesConforming) {
+        NSSet *subclassesOfSuper = [self subclassesOfClass:superclass];
+        [subclasses addObjectsFromArray:[subclassesOfSuper allObjects]];
+    }
+    
+    NSSet *allConforming = [superclassesConforming setByAddingObjectsFromSet:subclasses];
+    
+    return allConforming;
```

Comment:
That's a reasonable expectation, but if things aren't responding to `doesNotRespondToSelector:`, I have no faith that those things will reliably respond to `conformsToProtocol:`, which makes me want to stick to `class_conformsToProtocol()`.